docs/team/1szheng.md

```diff
@@ -3,8 +3,8 @@
 ## Project: PlanITarium
 
 PlanITarium is an application that assists you and your family in managing your finances, optimized for use on the
-Command Line Interface (CLI). You can use it to view your monthly financial status, logical group family members for
-better management and categorise your expenditure. PlanITarium is written in and meant to be run on `Java 11`.
+Command Line Interface (CLI). You can use it to view your monthly financial status, logically group family members for
+better management and categorise your expenditures. PlanITarium is written in and meant to be run on `Java 11`.
```

Comment:
Small English issue, `management and categorise` and be better split to `management, and categorise` for easier readability?